template.rb

```diff
@@ -8,8 +8,8 @@
 # This template assumes you have followed the setup guide at:
 # https://github.com/wildland/guides/#setting-up-your-development-enviroment
 #
-ruby_version = '2.1.5'
-node_version = 'iojs-v1.5.0'
+ruby_version = '2.2.2'
+node_version = 'v0.12.7'
```

Comment:
Referring to normal nodejs, v4 was the release that includes the merge of iojs into node: https://nodejs.org/en/blog/release/v4.0.0/.